.golangci.yml

```diff
@@ -293,11 +246,16 @@ issues:
     - path: pkg/networkservice/common/switchcase/.*_test.go
       linters:
         - dupl
-    - path: pkg/networkservice/chains/endpoint/combine_monitor_server.go
+    - path: pkg/tools/ippool/ippool.go
+      linters:
+        - unused
+    - path: pkg/networkservice/common/switchcase/common_test.go
+      linters:
+        - revive
+    - path: pkg/tools/opa/policies.go
       linters:
-        - interfacer
-      text: "`rawSrv` can be `github.com/networkservicemesh/sdk/pkg/networkservice/common/monitor.EventConsumer`"
+        - revive
+    # TODO - remove deprecated opentelemetry functions
```

Comment:
Please add a ticket to refer here. Thanks.

Reply:
Done